src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectOfStyles = {};
+  const arrStyles = sourceString.split(';');
+
+  const resultTrim = trimmer(arrStyles);
+
+  for (let i = 0; i < resultTrim.length; i++) {
+    if (resultTrim[i].length !== 0 && resultTrim[i][0].length !== 0) {
+      const style = resultTrim[i][0];
+      const value = resultTrim[i][1];
+
+      objectOfStyles[style.trim()] = value.trim();
+    }
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/js_style-to-object/blob/master/checklist.md) 5th item, let's use iteration methods for this. I suggest you using reduce to build object, with an initial value of `{}`